src/panels/lovelace/editor/hui-card-preview.ts

```diff
@@ -0,0 +1,57 @@
+import "@polymer/paper-input/paper-textarea";
+
+import createCardElement from "../common/create-card-element";
+import createErrorCardConfig from "../common/create-error-card-config";
+import { HomeAssistant } from "../../../types";
+import { LovelaceCard, LovelaceConfig } from "../types";
+import { ConfigError } from "./types";
+
+export class HuiCardPreview extends HTMLElement {
+  private _hass?: HomeAssistant;
+
+  set hass(value: HomeAssistant) {
+    this._hass = value;
+    if (this.lastChild) {
+      (this.lastChild as LovelaceCard).hass = value;
+    }
+  }
+
+  set error(error: ConfigError) {
+    const configValue = createErrorCardConfig(
+      `${error.type}: ${error.message}`,
+      undefined
+    );
+
+    this._createCard(configValue);
+  }
+
+  set value(configValue: LovelaceConfig) {
```

Comment:
We can make this `set config` right? To align with everything else as well as its param type